sdk/cognitiveservices/azopenai/go.mod

```diff
@@ -1,3 +1,4 @@
+// Deprecated: This package has moved to github.com/Azure/azure-sdk-for-go/sdk/ai/azopenai
```

Comment:
Nit, usually we just say "use <foo> instead"

Reply:
So "This has package has moved. Use github.com/Azure/azure-sdk-for-go/sdk/ai/azopenai instead."?

Reply:
I think you can omit the first sentence.  At any rate, take a look at what we did for the old keyvault modules and use the same verbiage so we're consistent.

Reply:
Changed to match what we put in keyvault.